x-pack/plugins/ml/common/types/detector_rules.ts

```diff
@@ -25,8 +26,17 @@ export interface DetectorRuleCondition {
   value: number;
 }
 
+export interface DetectorRuleParamsForceTimeShift {
+  time_shift_amount: long
```

Comment:
I think we can just use `number` here, using a `long` type won't have any meaningful effect, plus we don't use it anywhere else and so could cause type conflicts.

Reply:
Thank you for the review @jgowdyelastic . I changed it the `number` like you suggested.